src/services/checkIn.ts

```diff
@@ -61,7 +61,7 @@ type EnrollmentCheckin = {
   MinorHolds &
   Demographic;
 
-const getStatus = (): Promise<boolean> => http.get(`checkIn/status`);
+const getStatus = (): Promise<boolean> => http.get<boolean>(`checkIn/status`).catch(() => true);
```

Comment:
This is because the API returns a 404 error if no status is found (which is what happens when a student does not need to check in).

This is not for submitting their status, this is for checking whether they need to do it.

Reply:
> This is not for submitting their status, this is for checking whether they need to do it.

Yes, I knew that. My point was that if there was an error with checking if they needed to do the checkin, then they would never be required to complete it. Can we catch a 404 error specifically?